submissions/UA-girl/Task-HTML-CSS/index.html

```diff
@@ -0,0 +1,1538 @@
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="8 10"
+                  />
+                  <span class="booking-ico booking-ico__place">10</span>
+                  <span class="visually-hidden">8 ряд 10 місце</span>
+                </label>
+                <label class="booking-label booking-label__placebel">
+                  <input
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="8 11"
+                  />
+                  <span class="booking-ico booking-ico__place">11</span>
+                  <span class="visually-hidden">8 ряд 11 місце</span>
+                </label>
+                <label class="booking-label booking-label__placebel">
+                  <input
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="8 12"
+                  />
+                  <span class="booking-ico booking-ico__place">12</span>
+                  <span class="visually-hidden">8 ряд 12 місце</span>
+                </label>
+                <label class="booking-label booking-label__placebel">
+                  <input
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="8 13"
+                  />
+                  <span class="booking-ico booking-ico__place">13</span>
+                  <span class="visually-hidden">8 ряд 13 місце</span>
+                </label>
+                <label class="booking-label booking-label__placebel">
+                  <input
+                    type="checkbox"
+                    class="booking-input booking-input__place visually-hidden"
+                    data-available="1"
+                    data-price="100"
+                    value="8 14"
+                  />
+                  <span class="booking-ico booking-ico__place">14</span>
+                  <span class="visually-hidden">8 ряд 14 місце</span>
+                </label>
+              </fieldset>
+            </fieldset>
+            <button type="submit" class="button submit-button">
+              Підтвердити
+            </button>
+          </form>
+        </section>
+        <section class="order">
+          <h2 class="section-header order-header">Ваше замовлення</h2>
+          <table class="table order-table">
+            <tr>
+              <th class="table-header">Дата:</th>
+              <td class="table-data">30.01.2012p.</td>
```

Comment:
For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

Reply:
Thank you ) fixed